Source/Plugins/Core/com.equella.core/js/__stories__/components/MessageInfo.stories.tsx

```diff
@@ -16,75 +16,58 @@
  * limitations under the License.
  */
 import * as React from "react";
-import { action } from "@storybook/addon-actions";
-import { boolean, select, text } from "@storybook/addon-knobs";
-import MessageInfo from "../../tsrc/components/MessageInfo";
+import type { Meta, Story } from "@storybook/react";
+import MessageInfo, {
+  MessageInfoProps,
+} from "../../tsrc/components/MessageInfo";
 
 export default {
   title: "MessageInfo",
   component: MessageInfo,
-};
+  argTypes: { onClose: { action: "onClose" } },
+} as Meta<MessageInfoProps>;
 
-export const DynamicVariant = () => (
-  <MessageInfo
-    open={boolean("open", true)}
-    onClose={action("close")}
-    title={text(
-      "title",
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-    )}
-    variant={select(
-      "variant",
-      { success: "success", error: "error", info: "info", warning: "warning" },
-      "success"
-    )}
-  />
+export const DynamicVariant: Story<MessageInfoProps> = (args) => (
+  <MessageInfo {...args} />
 );
+DynamicVariant.args = {
+  open: true,
+  title: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+  variant: "success",
+};
 
-export const VariantSuccess = () => (
-  <MessageInfo
-    open={boolean("open", true)}
-    onClose={action("close")}
-    title={text(
-      "title",
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-    )}
-    variant="success"
-  />
+export const VariantSuccess: Story<MessageInfoProps> = (args) => (
+  <MessageInfo {...args} />
 );
+VariantSuccess.args = {
+  open: true,
+  title: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+  variant: "success",
+};
 
-export const VariantError = () => (
-  <MessageInfo
-    open={boolean("open", true)}
-    onClose={action("close")}
-    title={text(
-      "title",
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
-    )}
-    variant="error"
-  />
+export const VariantError: Story<MessageInfoProps> = (args) => (
+  <MessageInfo {...args} />
 );
+VariantError.args = {
+  open: true,
+  title: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+  variant: "error",
+};
```

Comment:
Part of the Storybook 6 Args is to promote re-use. So this should be:

```suggestion
VariantError.args = {
  ...VariantSuccess.args,
  variant: "error",
};
```

I guess ideally at the top we'd have a simple test, and then we'd possibly end-up with a less confusing name for this context. E.g. `Simple.args`.

This same suggestion is then the same for the below warning and info variants.

Reply:
Thanks for the feedback, refactored to share args